Packs/DatadogCloudSIEM/Integrations/DatadogCloudSIEM/DatadogCloudSIEM.yml

```diff
@@ -11,23 +11,31 @@ configuration:
 - additionalinfo: The API Key to use for authentication
   display: API Key
   name: api_key
-  required: true
-  type: 0
-- additionalinfo: The application key to use for authentication.
-  display: Application Key
+  type: 4
+  hidden: true
+- name: api_key_creds
+  type: 9
+  displaypassword: API Key
+  hiddenusername: true
+- display: Application Key
   name: app_key
-  required: true
-  type: 0
-- defaultvalue: '50'
-  display: How many incidents to fetch each time.
+  type: 4
+  additionalinfo: The application key to use for authentication.
+  hidden: true
+- name: app_key_creds
+  type: 9
+  displaypassword: Application Key
+  hiddenusername: true
+- display: How many incidents to fetch each time.
   name: max_fetch
   type: 0
   required: false
-- defaultvalue: 3 days
-  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3 months, 1 year)
+  defaultvalue: '50'
```

Comment:
why switch this?

Reply:
only the order was changed, not the values.